src/Compilers/VisualBasic/Portable/Symbols/Source/SynthesizedMyGroupCollectionPropertyAccessorSymbol.vb

```diff
@@ -227,15 +230,17 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
             '    <DisposeMethod>(Of <TargetType>)(<backingField>)
             ' End Set
 
-            Return "Set(ByVal " & StringConstants.ValueParameterName & " As " & targetTypeName & ")" & vbCrLf &
-                       "If " & StringConstants.ValueParameterName & " Is " & fieldName & vbCrLf &
-                           "Return" & vbCrLf &
-                       "End If" & vbCrLf &
-                       "If " & StringConstants.ValueParameterName & " IsNot Nothing Then" & vbCrLf &
-                           "Throw New Global.System.ArgumentException(""Property can only be set to Nothing"")" & vbCrLf &
-                       "End If" & vbCrLf &
-                       disposeMethodName & "(Of " & targetTypeName & ")(" & fieldName & ")" & vbCrLf &
-                   "End Set" & vbCrLf
+            Return _
+ $"Set(ByVal {StringConstants.ValueParameterName} As {targetTypeName})
```

Comment:
Left aligned as I think the indentation, could cause issues due significant whitespace within the string literal,